### DISCUSSION TITLE
This app bricks my device for 5 minutes on launch

### DISCUSSION THREAD

**Discussion by kokonut7** [bcssov/IronyModManager#470] [General]
It's a 2010 or something laptop, but still. Nothing else does. I'm playing hoi4, ck3 and rimworld on it.
P.S. nothing else is launched and the laptop is in high performance mode.

**Comment by bcssov** [comment]
It sounds like a potato PC, sorry. Best for you to uninstall the app.

**Reply by kokonut7** [reply]
So rude. As I've said, nothing else bricks it for like 5 minutes on launching. I just wanted to let you know it can cause that if run on a potato.

**Reply by bcssov** [reply]
Sure sure, I'm rude for telling you the truth. Irony is utilizing multithreading and parallel processing to speed up mod detection and analysis. Such an action does not "brick" up a computer, but if your computer is on the lower end (based on your example) it will appear to hiccup. It doesn't probably last 5 minutes but more mods you have more time it will take for it to analyze mods (it builds a realtime database of mods on each run). Based on your description you only have a single hard disk by the way too; so windows is dead in the water on such a computer in such a situation. Same happens with CK3 or any other paradox game or a similar application I reckon as well too.

I mean to put it into perspective it's like complaining about not being able to watch youtube videos using IE6.
